sdk/anomalydetector/azure-ai-anomalydetector/src/samples/java/com/azure/ai/anomalydetector/ReadmeSamples.java

```diff
@@ -3,9 +3,43 @@
 
 package com.azure.ai.anomalydetector;
 
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.http.policy.AzureKeyCredentialPolicy;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.ContentType;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpHeaders;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.HttpPipelineBuilder;
+import com.azure.core.http.policy.AddHeadersPolicy;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+
 public final class ReadmeSamples {
-    public void readmeSamples() {
-        // BEGIN: com.azure.ai.anomalydetector.readme
-        // END: com.azure.ai.anomalydetector.readme
+
+    private static void createClient() {
+        // BEGIN: readme-sample-createAnomalyDetectorClient
+        String endpoint = "<anomaly-detector-resource-endpoint>";
+        String key = "<anomaly-detector-resource-key>";
+
+        HttpHeaders headers = new HttpHeaders()
+            .put("Accept", ContentType.APPLICATION_JSON);
```

Comment:
Is adding the "accept" header necessary?

Reply:
follow the previous version.

Reply:
Is this required for creating the client?

Reply:
Usually it is already set in code https://github.com/Azure/azure-sdk-for-java/blob/cadl_dpg_anomalydetector/sdk/anomalydetector/azure-ai-anomalydetector/src/main/java/com/azure/ai/anomalydetector/implementation/AnomalyDetectorClientImpl.java#L453